src/SFML/Audio/SoundStream.cpp

```diff
@@ -395,6 +507,14 @@ bool SoundStream::getLoop() const
 }
 
 
+////////////////////////////////////////////////////////////
+void SoundStream::setEffectProcessor(EffectProcessor effectProcessor)
+{
+    m_impl->effectProcessor = std::move(effectProcessor);
+    m_impl->connectEffect(!!m_impl->effectProcessor);
+}
```

Comment:
How does this API respond to passing in a null `std::function` object? Does that simply disable effect processing?

Reply:
`effectProcessor` is set to `{}` and `connectEffect` is called with `false` resulting in the effect node being removed from the node graph. Without it being in the node graph it is completely bypassed.